typescript/index.d.ts

```diff
@@ -250,6 +255,26 @@ interface RaygunStatic {
      * Track Single Page Application route events.
      */
     trackEvent(type: "pageView", options: { path: string }): void;
+
+    /**
+     * Records a manual breadcrumb with the given message and metadata passed.
+     */
+    recordBreadcrumb(message:string,metadata?:any): void;
```

Comment:
@BenjaminHarding this should actually be `message:string|any` I think. You can either call `recordBreadcrumb("message", metadata)` or `recordBreadcrumb({anEntireBreadcrumbObject})`